pkg/apb/bind.go

```diff
@@ -22,17 +22,13 @@ func Bind(instance *ServiceInstance, parameters *Parameters, clusterConfig Clust
 	log.Notice(fmt.Sprintf("ServiceInstance.Description: %s", instance.Spec.Description))
 	log.Notice("============================================================")
 
-	var client *Client
-	var err error
-
-	if client, err = NewClient(log); err != nil {
-		return "", nil, err
-	}
-
-	podName, err := client.RunImage("bind", clusterConfig, instance.Spec, instance.Context, parameters)
+	podName, err := ExecuteApb(
+		"bind", clusterConfig, instance.Spec,
+		instance.Context, parameters, log,
+	)
 
 	if err != nil {
-		log.Error("Problem running image", err)
+		log.Error("Problem executing apb: %s", err)
```

Comment:
We will print out the error message later on, but we won't have podName.  I think it would be more beneficial to print out podName here instead of err. Copy from provision.go